pgmgr/pgmgr.go

```diff
@@ -335,6 +349,10 @@ func applyMigrationByPsql(c *Config, m Migration, direction int) error {
 	}
 	defer os.Remove(tmpfile.Name()) // clean up
 
+	if _, err := tmpfile.WriteString(timeoutStatements(c)); err != nil {
```

Comment:
Do the timeouts need to be reset after each migration?

Reply:
Yes, since each migration runs through a separate invocation of psql and thus a new connection.

Reply:
On the pq driver, when the connection doesn't change, not necessarily -- but it does prevent any in-migration overrides from bleeding through to the next migration.